tests/e2e/actions.go

```diff
@@ -1974,6 +1974,19 @@ func (tr TestRun) waitForSlashMeterReplenishment(
 	}
 }
 
+type slashPacketRetryAction struct {
+	consumer chainID
+}
+
+func (tr TestRun) waitForSlashRetry(
+	action slashPacketRetryAction,
+	verbose bool,
+) {
+	// Retry delay period is set to 200 seconds, see config.go
+	// wait this amount of time to elapse the period
+	tr.WaitTime(200 * time.Second)
```

Comment:
Well, the `slashMeterReplenishmentAction` takes some amount of time, usually 60 seconds ish. I set the retry delay period to 200 seconds so that the retry delay period would end during the appropriate action (ie. it would not end during `slashMeterReplenishmentAction`. However, the steps still pass if I set everything to 30 seconds, so no problem from my end

https://github.com/cosmos/interchain-security/commit/d089c5186f2fd0f24b198c7889a8ea040464b339